lib/full-site-editing/block-templates.php

```diff
@@ -377,32 +394,36 @@ function gutenberg_get_block_template( $id, $template_type = 'wp_template' ) {
 		return null;
 	}
 	list( $theme, $slug ) = $parts;
-	$wp_query_args        = array(
-		'post_name__in'  => array( $slug ),
-		'post_type'      => $template_type,
-		'post_status'    => array( 'auto-draft', 'draft', 'publish', 'trash' ),
-		'posts_per_page' => 1,
-		'no_found_rows'  => true,
-		'tax_query'      => array(
-			array(
-				'taxonomy' => 'wp_theme',
-				'field'    => 'name',
-				'terms'    => $theme,
-			),
-		),
-	);
-	$template_query       = new WP_Query( $wp_query_args );
-	$posts                = $template_query->get_posts();
 
-	if ( count( $posts ) > 0 ) {
-		$template = _gutenberg_build_template_result_from_post( $posts[0] );
+	$active = wp_get_theme()->get_stylesheet() === $theme;
+
+	// Temporarily disable inactive access for 5.8 version.
+	if ( ! $active ) {
+		return null;
+	}
+
+	if ( $active ) {
```

Comment:
We can remove this check, and the other branches.

Reply:
Done.